### PR TITLE
Happymh: Fix page list

### LINKED ISSUE #6273: Happymh could not pass cloudflare
### Source information

Happymh 1.4.13

### Source language

Chinese

### Steps to reproduce

1. choose webview 
2. Pass webview cloudflare test
3. Back to chapter
4. Choose chapter 
5. Back screen and keep loading 
6. Show could not pass cloudflare message 

### Expected behavior

Should show galleries 

### Actual behavior

Back screen and keep loading 

### Mihon/Tachiyomi version

0.15.3

### Android version

14

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### 

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/keiyoushi/extensions-source/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by anenasa**
Closes #6273

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [ ] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

**Comment by BabyNurse020517** [comment]
my happymh still not working. it show 503 service temporarily unavailable. is this just me or ?

**Comment by Smol-Ame** [comment]
@BabyNurse020517 this is an open/proposed pull request that is pending review & approval before it is merged into the repo (afterwards the extension will get an update when it's merged & built).

Don't comment if you don't have any details or info that will help a contributor with finding/creating a fix.

**Review by vetleledaal — APPROVED** [review]

**Pull request merged**
